packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts

```diff
@@ -40,6 +40,22 @@ withChainsDescribe("Tokens reducer", () => {
           tokenTicker: "ETH",
         },
       },
+      TRASH: {
+        chainId: "ethereum-eip155-5777",
+        token: {
+          fractionalDigits: 18,
+          tokenName: "Trash Token",
+          tokenTicker: "TRASH",
+        },
+      },
+      ASH: {
+        chainId: "ethereum-eip155-5777",
+        token: {
+          fractionalDigits: 18,
```

Comment:
This must be 12, see https://github.com/iov-one/ethereum-smart-contracts/blob/master/contracts/AshToken.sol